hapi/plugin-webpack.js

```diff
@@ -14,6 +15,7 @@ function bundle(server, options, next) {
   var done = _.after(bundleCounts, function() {
     logger.info(statistics[0]);
     logger.ok('Finished bundling');
+    context.emit('av:bundle-complete');
```

Comment:
I'm not sure if you have a preferred format/namespace for events.  So let me know if you want this to be something else.